apps/condo/domains/acquiring/schema/Payment.js

```diff
@@ -160,10 +160,14 @@ const Payment = new GQLListSchema('Payment', {
             isRequired: false,
             kmigratorOptions: { null: true, on_delete: 'models.SET_NULL' },
             hooks: {
-                validateInput: ({ resolvedData, addFieldValidationError, fieldPath }) => {
-                    if (resolvedData[fieldPath] && !resolvedData['frozenReceipt']) {
+                validateInput: ({ operation, existingItem, resolvedData, addFieldValidationError, fieldPath }) => {
+                    // We need to connect receipt, if it was not present on creation
```

Comment:
Seems strange, how can I pay for nothing?

Reply:
RegisterMultiPaymentForVirtualReceiptService. Mostly it's payments with qr-code, where bank data present, but no link for receipt in our system. Also resident can overpay, just pay by same bank data before organization published receipt